compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1334,6 +1333,16 @@ static LogicalResult setRootConfig(func::FuncOp entryPointFn,
   return setConvRootConfig(entryPointFn, convOp, targetTileSizes, vectorSize);
 }
 
+/// Sets the lowering configuration for linalg.conv_2d_nchw_fchw
+/// operations.
+static LogicalResult setRootConfig(func::FuncOp entryPointFn,
+                                   linalg::Conv2DNchwFchwOp convOp) {
+  int64_t vectorSize =
+      getVectorSize(entryPointFn, convOp.getResult(0).getType());
+  SmallVector<int64_t> targetTileSizes = {1, vectorSize * 2, 1, 8, 8, 1, 1};
```

Comment:
Are the tiling sizes arbitrary values? Or they are roughly tuned?

This needs to be refactored to use `getConvWorkgroupSizes`. I have a WIP commit and I can fix it there. You don't have to fix it in the PR.

Reply:
yeap, these are tuned from the previous KernelDispatch for nhwc before getConvWorkgroupSizes was implemented, but I remapped the tile sizes to work with nchw.